web-ng/src/app/components/layer-dialog/layer-dialog.component.html

```diff
@@ -13,10 +13,18 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-
+<span> Default Style: </span>
```

Comment:
Nit: Remove leading and trailing spaces.

Reply:
Done :)